### PR TITLE
Adds recent activity for items to admin page.

### LINKED ISSUE #2325: Display recent activity for items on Admin page
THIS IS ONLY FOR ITEMS ACTIVITY; FOR COLLECTIONS ACTIVITY SEE #2446

This table should display all the time on the admin page that is accessible only to administrators, under the dropdown menu for the other functionality.

Mockup: https://projects.invisionapp.com/share/EQXC9CLKCR2#/screens/467473848

Screen Shot 2022-06-16 at 3 53 45 PM

After the table header it should indicate the three available options of recent activity for last "7 days | 14 days | 30 days."

Types of activity to be included in the items table:
- updated by user
- submitted for review
- submitted for deposit
- deposit completed
- item locked
- item unlocked
- item decommissioned
- ownership changed

This table should be collapsible, and it should be collapsed by default.

Item table should include the following columns:
- item title, linked to item view page
- collection title, linked to collection view page
- type of activity
- date/time of activity

### PR DESCRIPTION
**Pull request by justinlittman**
closes #2325

## Why was this change made? 🤔
Per ticket.

## How was this change tested? 🤨

⚡ ⚠ If this change involves consuming from or writing to another service (or shared file system), ***run [integration test create_object_h2_spec.rb](https://github.com/sul-dlss/infrastructure-integration-test)*** and/or test manually in [stage|qa] environment, in addition to specs. ⚡

Unit

### CODE REVIEW

**Comment by justinlittman** [comment]
image

File: .rubocop.yml [hunk 1]
```diff
@@ -64,6 +64,10 @@ RSpec/ScatteredSetup:
   Exclude:
     - spec/policies/**/*
 
+RSpec/LetSetup:
+  Exclude:
+    - spec/requests/admins_spec.rb
```
Comment (jcoyne) [inline .rubocop.yml:69]: Please move this to an inline exception.

File: app/components/admin/items_activity_component.html.erb [hunk 2]
```diff
@@ -0,0 +1,10 @@
+<section class="mb-5">
+  <button class="btn collapse-header" data-bs-toggle="collapse" data-bs-target="#itemsActivity" role="button" aria-expanded="false" aria-controls="itemsActivity">
+    Items recent activity
+  </button>
+  <div class="collapse" id="itemsActivity">
+    <turbo-frame id="itemsActivityFrame" src="<%= items_recent_activity_admin_path%>">
```
Comment (jcoyne) [inline app/components/admin/items_activity_component.html.erb:6]: would you mind adding an extra space here before `%>`

File: spec/requests/admins_spec.rb [hunk 3]
```diff
@@ -9,8 +9,10 @@
     let(:work_collection) { create(:collection, creator: user) }
     let(:work1) { create(:work, owner: user, collection: work_collection) }
     let(:work_version1) { create(:work_version, state: 'deposited', work: work1) }
+    let!(:event1) { create(:embargo_lifted_event, eventable: work1, created_at: 3.days.ago) }
```
Comment (jcoyne) [inline spec/requests/admins_spec.rb:12]: This never seems to be used. Can it just go in the `before` block?

File: spec/requests/admins_spec.rb [hunk 4]
```diff
@@ -9,8 +9,10 @@
...
     let(:work2) { create(:work, owner: user, collection: work_collection) }
     let(:work_version2) { create(:work_version, state: 'first_draft', work: work2) }
+    let!(:event2) { create(:event, eventable: work2) }
```
Comment (jcoyne) [inline spec/requests/admins_spec.rb:15]: and this one too

**Review by jcoyne — CHANGES_REQUESTED** [review]

File: app/components/admin/item_activity_row_component.html.erb [hunk 5]
```diff
@@ -0,0 +1,6 @@
+<tr>
+  <td><%= link_to item_title, item, target: '_top' %></td>
```
Comment (jcoyne) [inline app/components/admin/item_activity_row_component.html.erb:2]: what is `target: '_top'` doing for us here?  Should it be documented?

**Review by jcoyne — APPROVED** [review]
Approved pending consideration of comment.

File: spec/requests/admins_spec.rb [hunk 6]
```diff
@@ -14,18 +14,31 @@
 
     before do
       create(:collection_version_with_collection, collection: work_collection)
+      create(:embargo_lifted_event, eventable: work1, created_at: 3.days.ago)
```
Comment (jcoyne) [inline spec/requests/admins_spec.rb:17]: we might want these lines only in the scope that is for items_recent_activity or they get created even on requests where they're not used.

**Pull request merged**